docs/apis/cli.md

```diff
@@ -252,6 +256,12 @@ Action "list" lists running and scheduled programs.
                                    configuration.
      -r,--running                  Show only running programs and their JobIDs
      -s,--scheduled                Show only scheduled programs and their JobIDs
+  Additional arguments if -m yarn-cluster is set:
+     -yid <yarnApplicationId>      YARN application ID of Flink YARN session to
+                                   connect to. Must not be set if JobManager HA
+                                   is used. In this case, JobManager RPC
+                                   location is automatically retrieved from
+                                   Zookeeper.
```

Comment:
I tried this, but I wonder why its not logging that the job has been cancelled. 
It logs that "All TaskManagers are connected", I don't think this message is relevant when cancelling a job.

```
[cloudera@quickstart build-target]$  ./bin/flink cancel -m yarn-cluster -yid application_1447844011707_0038 b9b8f76616073d09c596545a3cda978f
2016-06-13 07:22:21,355 INFO  org.apache.hadoop.yarn.client.RMProxy                         - Connecting to ResourceManager at quickstart.cloudera/10.0.2.15:8032
2016-06-13 07:22:22,277 INFO  org.apache.flink.yarn.YarnClusterClient                       - Start application client.
2016-06-13 07:22:22,288 INFO  org.apache.flink.yarn.ApplicationClient                       - Notification about new leader address akka.tcp://flink@10.0.2.15:51747/user/jobmanager with session ID null.
2016-06-13 07:22:22,290 INFO  org.apache.flink.yarn.ApplicationClient                       - Received address of new leader akka.tcp://flink@10.0.2.15:51747/user/jobmanager with session ID null.
2016-06-13 07:22:22,290 INFO  org.apache.flink.yarn.ApplicationClient                       - Disconnect from JobManager null.
Waiting until all TaskManagers have connected
2016-06-13 07:22:22,297 INFO  org.apache.flink.yarn.ApplicationClient                       - Trying to register at JobManager akka.tcp://flink@10.0.2.15:51747/user/jobmanager.
No status updates from the YARN cluster received so far. Waiting ...
2016-06-13 07:22:22,542 INFO  org.apache.flink.yarn.ApplicationClient                       - Successfully registered at the ResourceManager using JobManager Actor[akka.tcp://flink@10.0.2.15:51747/user/jobmanager#1733798764]
All TaskManagers are connected
2016-06-13 07:22:22,945 INFO  org.apache.flink.yarn.YarnClusterClient                       - Shutting down YarnClusterClient from the client shutdown hook
2016-06-13 07:22:22,945 INFO  org.apache.flink.yarn.YarnClusterClient                       - Disconnecting YarnClusterClient from ApplicationMaster
2016-06-13 07:22:22,947 INFO  org.apache.flink.yarn.ApplicationClient                       - Stopped Application client.
2016-06-13 07:22:22,947 INFO  org.apache.flink.yarn.ApplicationClient                       - Disconnect from JobManager Actor[akka.tcp://flink@10.0.2.15:51747/user/jobmanager#1733798764].
2016-06-13 07:22:23,056 INFO  org.apache.flink.yarn.YarnClusterClient                       - Application application_1447844011707_0038 finished with state RUNNING and final state UNDEFINED at 0
[cloudera@quickstart build-target]$ 
```